core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -110,9 +110,8 @@ public enum PropertyKey implements ConfigPropertyKey {
     CAIRO_SQL_WINDOW_INITIAL_RANGE_BUFFER_SIZE("cairo.sql.window.initial.range.buffer.size"),
     CAIRO_SQL_ANALYTIC_COLUMN_POOL_CAPACITY("cairo.sql.analytic.column.pool.capacity"),
     CAIRO_SQL_WINDOW_COLUMN_POOL_CAPACITY("cairo.sql.window.column.pool.capacity"),
-    CAIRO_SQL_CREATE_TABLE_MODEL_POOL_CAPACITY("cairo.sql.create.table.model.pool.capacity"),
     CAIRO_SQL_CREATE_TABLE_MODEL_BATCH_SIZE("cairo.sql.create.table.model.batch.size"),
-    CAIRO_SQL_COLUMN_CAST_MODEL_POOL_CAPACITY("cairo.sql.column.cast.model.pool.capacity"),
+    CAIRO_SQL_CREATE_TABLE_COLUMN_MODEL_POOL_CAPACITY("cairo.create.table.column.model.pool.capacity"),
```

Comment:
at very least the PR should be marked as breaking and changes should be explained to users can act on it.

Reply:
thanks Jaromir, this is super obscure param, i bet no one uses it because no one would understand what's it for. I would not alarm users. What do you think?

Reply:
I guess you are right. still, maybe worth registering it in https://github.com/questdb/questdb/blob/4924b01431ab019630173a5a773299055e24045b/core/src/main/java/io/questdb/PropServerConfiguration.java#L1917-L1921 ? so if there is a user who set this explicitly they will get a a warning? also - please change it https://github.com/questdb/questdb/blob/a4b9a7e40884bac3989fbb1f3c344a481387241d/core/src/main/resources/io/questdb/site/conf/server.conf#L341 and in https://github.com/questdb/questdb/blob/92eff1e713c8586e5f88f4e3ea2790efa8c32997/pkg/ami/marketplace/assets/server.conf#L311

Reply:
registered the param as deprecated here: https://github.com/questdb/questdb/pull/5159/commits/444ab4c40d85284b1d945d81ffa163be2db587fc